test/integration/test_integration.py

```diff
@@ -60,7 +60,7 @@ def test_two_three_nevr_passed(nevr):
     assert run_task(nevr)['python-versions.two_three'].outcome == 'PASSED'
 
 
-@pytest.fixture()
+@pytest.fixture(scope="session")
```

Comment:
Please do. I am not familiar with pytest fixture scopes much, but I would like to see how this will work,